tools/cockpit.spec

```diff
@@ -94,9 +94,11 @@ BuildRequires: sed
 BuildRequires: xmlto
 
 Requires: %{name}-bridge = %{version}-%{release}
+Requires: %{name}-networkmanager = %{version}-%{release}
 Requires: %{name}-ws = %{version}-%{release}
 Requires: %{name}-shell = %{version}-%{release}
 Requires: %{name}-storaged = %{version}-%{release}
+Requires: %{name}-users = %{version}-%{release}
```

Comment:
I would suggest keeping the user stuff in the shell RPM. It doesn't have any big dependencies.